src/Symfony/Component/Serializer/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ CHANGELOG
  * added an optional `string $format = null` argument to `AbstractNormalizer::instantiateObject`
  * added an optional `array $context = array()` to `Serializer::supportsNormalization`, `Serializer::supportsDenormalization`,
    `Serializer::supportsEncoding` and `Serializer::supportsDecoding`
+ * added optional `bool $escapeFormulas = false` argument to `CsvEncoder::__construct`
```

Comment:
should be moved to a new 4.1.0 section
also please mind @ostrolucky's comment below